CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - On Windows, query DirectWrite for recommended anti-aliasing settings
+- On Windows, the ConPTY backend will now be used by default if available; the `enable_experimental_conpty_backend` config option has been replaced with `use_winpty_backend`.
```

Comment:
I think it might be easier to split this off a bit, that way we don't need multiple sentences and run into huge lines.

We also don't use any punctuation (other than commas) in change log entries.

I'd recommend to have one entry in `Changed` which points out that `ConPTY` is now enabled by default and a separate entry that points out that `enable_experimental_conpty_backend` has been replaced with `use_winpty_backend`.